nixos/modules/services/networking/yggdrasil.nix

```diff
@@ -181,7 +188,7 @@ in
             "${binYggdrasil} -genconf") + " > /run/yggdrasil/yggdrasil.conf"}
 
           # start yggdrasil
-          ${binYggdrasil} -useconffile /run/yggdrasil/yggdrasil.conf
+          ${binYggdrasil} -useconffile /run/yggdrasil/yggdrasil.conf ${toString cfg.extraArgs}
```

Comment:
Is it better to use `escapeShellArgs`?

Reply:
Yes. That will also handle spaces and other stuff inside arguments.